birdhouse/env.local.example

```diff
@@ -17,7 +20,6 @@ export TWITCHER_PROTECTED_PATH=/twitcher/ows/proxy
 export PHOENIX_PASSWORD=phoenix_pass
 export PHOENIX_PASSWORD_HASH=sha256:123456789012:1234567890123456789012345678901234567890123456789012345678901234
 export TOMCAT_NCWMS_PASSWORD=ncwmspass
-export SUPPORT_EMAIL=helpdesk@example.com
```

Comment:
Can you restore this `SUPPORT_EMAIL` here since it was previously moved to `default.env` and then you reverted that move (agree with the revert by the way, thanks).

Reply:
done